Blankly/interface/abc_currency_interface.py

```diff
@@ -192,16 +193,21 @@ def get_fees(self) -> dict:
 
     """
     @abc.abstractmethod
-    def history(self,
-                product_id: str,
-                to: Union[str, int, float],
-                granularity: Union[str, float]) -> pandas.DataFrame:
+    def history(self, 
+                product_id: str, 
+                to: Union[str, int]=200, 
```

Comment:
We need float here because time.time() is a float and binance gives floats.

Reply:
This is a little bit different namely the fact that the to is either a string or the number of points backwards by a spec ific resolution, check out how `epoch_start` is calculated

Reply:
sure but if I pass in a value returned by binance then my IDE will get mad.